CHANGELOG.md

```diff
@@ -22,6 +22,10 @@ For information on changes in released versions of Teku, see the [releases page]
 - Docker images are now being published to `consensys/teku`. The `pegasys/teku` images are no longer updated.
 
 ### Additions and Improvements
+- New validators can now be loaded without restarting by sending a `SIGHUP` to Teku. 
+  Teku will recheck directories of keys specified via `--validator-keys` and URLs specified via `--validators-external-signer-public-keys` for any new validators to run.
+  
+  *Important*: Teku does not support removing validators via this method as there is no reliable mechanism to signal when they have actually ceased performing duties.
```

Comment:
Do we want to state 'Removing validators will therefore still require a restart of teku' explicitly?

Reply:
Seems reasonable.